samples/textcompletion/csharp-ooproc/TextCompletions.cs

```diff
@@ -2,9 +2,9 @@
 // Licensed under the MIT License.
 
 using System.Net;
+using Functions.Worker.Extensions.OpenAI.TextCompletion;
```

Comment:
This doesn't look right. All our namespaces should start with `Microsoft.Azure`.

Reply:
UPdated!